src/lib/node/message_handler.ts

```diff
@@ -211,6 +211,9 @@ export class NodeMessageHandler {
       case NodeCommand.waitForWakeup:
         await node.waitForWakeup();
         return {};
+      case NodeCommand.interview:
+        node.interview();
```

Comment:
should this be awaited?

Reply:
The docs suggest not waiting for the interview to complete because of how long it can take

Reply:
I think that we should await it. It's up to the caller (Home Assistant) to decide if they want to await the task or not.

Reply:
should we do this in other places too? right now we don't await `node.interviewCC` for the same reason

Reply:
I think so.